sim/dex-data.ts

```diff
@@ -40,8 +40,8 @@ export class Tools {
 	static getId(text: any): ID {
 		if (text && text.id) {
 			text = text.id;
-		} else if (text && text.userid) {
```

Comment:
I'm not sure if it matters anywhere but we might want to keep the .userid check

Reply:
I guess it's still used for `muteQueue`, `BattleReady`, etc. Re-added.